docs/pages/bare/installing-updates.mdx

```diff
@@ -65,37 +65,218 @@ Add the **"Supporting"** directory containing **Expo.plist** to your project in
+Apply the following configurations to your **AndroidManifest.xml** file:
 
-<DiffBlock
-  raw={`
-diff --git a/apps/bare-update/android/app/src/main/AndroidManifest.xml b/apps/bare-update/android/app/src/main/AndroidManifest.xml
---- a/apps/bare-update/android/app/src/main/AndroidManifest.xml
-+++ b/apps/bare-update/android/app/src/main/AndroidManifest.xml
-@@ -5,6 +5,8 @@
-   <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
-   <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
-   <application android:name=".MainApplication" android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:allowBackup="false" android:theme="@style/AppTheme" android:usesCleartextTraffic="true">
-+    <meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="https://exp.host/@my-expo-username/my-app"/>
-+    <meta-data android:name="expo.modules.updates.EXPO_SDK_VERSION" android:value="45.0.0"/>
-     <activity android:name=".MainActivity" android:label="@string/app_name" android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode" android:launchMode="singleTask" android:windowSoftInputMode="adjustResize" android:theme="@style/Theme.App.SplashScreen">
-       <intent-filter>
-         <action android:name="android.intent.action.MAIN"/>
-`}
-/>
+```diff AndroidManifest.xml
++ <meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="https://exp.host/@my-expo-username/my-app" />
++ <meta-data android:name="expo.modules.updates.EXPO_SDK_VERSION" android:value="46.0.0" />
++ <meta-data android:name="expo.modules.updates.UPDATES_CONFIGURATION_REQUEST_HEADERS_KEY" android:value="{'expo-channel-name':'your-channel-name'}"/>
++ <meta-data android:name="expo.modules.updates.EXPO_UPDATES_ENABLED" android:value="true" />
++ <meta-data android:name="expo.modules.updates.EXPO_UPDATES_LAUNCH_WAIT_MS" android:value="0" />
++ <meta-data android:name="expo.modules.updates.EXPO_UPDATES_CHECK_ON_LAUNCH" android:value="ALWAYS" />
+````
 
 ## Customizing Automatic Setup for iOS
 
 By default, `expo-updates` requires no additional setup. If you want to customize the installation, e.g. to enable updates only in some build variants, you can instead follow these manual setup steps and then apply any customizations.
 
-<DiffBlock source="/static/diffs/expo-updates-ios.diff" />
+### Update AppDelegate.h
+
+```diff apps/bare-update/ios/bareupdate/AppDelegate.h
+#import <Foundation/Foundation.h>
++ #import <EXUpdates/EXUpdatesAppController.h>
+#import <React/RCTBridgeDelegate.h>
+#import <UIKit/UIKit.h>
+
+#import <ExpoModulesCore/EXAppDelegateWrapper.h>
+
+- @interface AppDelegate : EXAppDelegateWrapper <RCTBridgeDelegate>
++ @interface AppDelegate : EXAppDelegateWrapper <RCTBridgeDelegate, EXUpdatesAppControllerDelegate>
+```
+
+### Update AppDelegate.mm
+
+There are multiple changes to apply to your project's AppDelegate.mm.
```

Comment:
```suggestion
There are multiple changes to apply to your project's **AppDelegate.mm**.
```

Since it's a file name, we should make it bold [as per our styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text).